recipes/jupyterlite-sphinx/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "jupyterlite-sphinx" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlite_sphinx-{{ version }}.tar.gz
+  sha256: f2ae7f05202ed9edbeb2a0e84746b836672ab1436e062f48a48dfd288fdfd14f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - hatchling
+    - pip
+  run:
+    - python >=3.8
+    - docutils
+    - jupyter_server
+    - jupyterlab_server
+    - jupyterlite-core >=0.1.0,<0.2.0
+    - sphinx >=4
+
+test:
+  imports:
+    - jupyterlite_sphinx
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jupyterlite/jupyterlite-sphinx
+  summary: Sphinx extension for deploying JupyterLite
+  doc_url: https://jupyterlite-sphinx.readthedocs.io
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jtpio
+    - martinRenou
```

Comment:
cc @martinRenou if you agree to be listed here

Reply:
Yes! Thanks!